js/main.js

```diff
@@ -80,19 +107,42 @@ function switchViews(view) {
       data.view = $viewElements[i].getAttribute('data-view');
     }
   }
+  if ($ul.childElementCount === 0) {
+    $noItems.className = 'center-text';
+  } else {
+    $noItems.className = 'center-text hidden';
+  }
+
 }
 
 window.addEventListener('DOMContentLoaded', appendDOM);
 var $noItems = document.querySelector('.center-text');
-if ($ul.childElementCount === 0) {
-  $noItems.className = 'center-text hidden';
-} else {
-  $noItems.className = 'center-text';
-}
+
 
 function appendDOM() {
   for (var i = 0; i < data.entries.length; i++) {
     entryDOM(data.entries[i]);
   }
   switchViews(data.view);
 }
+
+
+$entriesPage.addEventListener('click', function () {
+  
+  if (event.target.tagName === 'I') {
+      var $icon = document.querySelectorAll('i');
+      for (var i = 0; i < $icon.length; i++) {
+        $icon[i].setAttribute("data-entry-id", $icon.length - i);
+      }
+      switchViews(event.target.getAttribute('data-view'))
+    for (var x = 0; x < data.entries.length; x++){
+      if (data.entries[x].entryId === parseInt(event.target.attributes['data-entry-id'].value)) {
+        $inputTitle.value = data.entries[x].title;
+        $imgUrl.value = data.entries[x].photoUrl
+        $textArea.value = data.entries[x].notes
+        $setImg.setAttribute('src', data.entries[x].photoUrl);
+        data.editing = parseInt(event.target.attributes['data-entry-id'].value)
```

Comment:
Instead of saving just the id in `data.editing`, I suggest saving the entire object for the entry